src/actions/actions.ts

```diff
@@ -1094,12 +1094,12 @@ class CommandDot extends BaseCommand {
 
 @RegisterAction
 class CommandFold extends BaseCommand {
-  modes = [ModeName.Visual, ModeName.VisualLine];
+  modes = [ModeName.Normal];
```

Comment:
I suppose `zc` also works in Visual/Visual Line mode, so we just need to add Normal mode to the list.